app.rb

```diff
@@ -3,47 +3,59 @@
 require './school_student'
 require './school_teacher'
 require './school_book'
-
+require './json'
```

Comment:
- [ ] the application is not running due to the `require './json'` in this line, please change this to `require 'json'`.
- [ ] however when removing this line there are still other error's the user will run into, please make sure to fix all errors, before requesting a review.